### PR TITLE
Implemented GET operation to obtain all the history of an user

### PR DESCRIPTION
**Pull request by JavierMotaAT**

### CODE REVIEW

File: src/main/java/com/agilethought/atsceapi/controller/HistoryController.java [hunk 1]
```diff
@@ -18,4 +28,10 @@
     public NewHistoryResponse postHistory(@RequestBody NewHistoryRequest request) {
         return historyService.createHistory(request);
     }
+    
+	@ResponseStatus(HttpStatus.OK)
+	@GetMapping("/histories")
```
Comment (luisgalindoluna) [inline src/main/java/com/agilethought/atsceapi/controller/HistoryController.java:33]: Isn't your endpoint supposed to be "**histories?userid={id}**"?

**Review by luisgalindoluna — CHANGES_REQUESTED** [review]
Very well done, except you just have to make sure you are using the endpoint you were provided in your US.

**Review by luisgalindoluna — APPROVED** [review]
Looks good

File: src/main/java/com/agilethought/atsceapi/service/HistoryServiceImpl.java [hunk 2]
```diff
@@ -37,4 +41,10 @@ public NewHistoryResponse createHistory(NewHistoryRequest request) {
         History savedHistory = historyRepository.save(newHistory);
         return new NewHistoryResponse(savedHistory.getId());
     }
+    
+	@Override
+	public List<History> getAllHistory(String id) {
```
Comment (ivanflores15) [inline src/main/java/com/agilethought/atsceapi/service/HistoryServiceImpl.java:46]: if I sent `/api/v1/histories?userId=` , what is the response?

Comment (JavierMotaAT) [reply src/main/java/com/agilethought/atsceapi/service/HistoryServiceImpl.java:46]: An empty array

**Review by ivanflores15 — APPROVED** [review]
Greate! looks good

**Comment by ivanflores15** [comment]
once you code is merged, please deploy your code using jenkins pipeline.

**Review by EmiliaGuillen-dev — APPROVED** [review]
looks great

**Pull request merged**